doc/source/index.rst

```diff
@@ -4,23 +4,107 @@
 
 .. include:: include/global.rst
 
-Welcome to the documentation of the Python interface of igraph
-==============================================================
+.. currentmodule:: igraph
+
+.. raw:: html
+
+    <style type="text/css">
+    div.twocol {
+        padding-left: 0;
+        padding-right: 0;
+        display: flex;
+        gap: 20px;
```

Comment:
The `gap` property is not supported on certain older browsers (see [here](https://caniuse.com/flexbox-gap)) and it's not supported on IE11 either. Since we know that we are going to have two columns only, you could use `margin-right: 20px;` on `div.twocol > div` instead, or add a 10px horizontal padding on both divs and then add a negative 10px left margin on the first div and a negative 10px right margin on the second div if you want to prepare for multiple columns.

Or, use `gap` in grid layout instead of flexbox; `gap` has better support in grid layout (see [here](https://caniuse.com/mdn-css_properties_gap_grid_context) ).